specs-go/chain/chainid.go

```diff
@@ -0,0 +1,46 @@
+package chain
+
+import "github.com/docker/distribution/digest"
+
+// ChainID takes a slice of digests and returns the ChainID corresponding to
+// the last entry. Typically, these are a list of layer DiffIDs, with the
```

Comment:
I'd rather drop “to the last entry”.  The ChainID is a property of [the slice as a whole][1], not a property of one entry in particular.

[1]: https://github.com/opencontainers/image-spec/issues/482#issuecomment-265565694